simplified-accounts-api/src/main/java/org/nypl/simplified/accounts/api/AccountAuthenticationAdobePreActivationCredentials.kt

```diff
@@ -36,7 +36,7 @@ data class AccountAuthenticationAdobePreActivationCredentials(
   val deviceManagerURI: URI?,
 
   /**
-   * The post activation credentials that resulted from device activation, iff any device activation has been performed
```

Comment:
Actually, this is short for "if and only if".

"If" means "If there are credentials → Device activation has been performed." but doesn't necessarily imply "If device activation has been performed → There are credentials".

"Iff" means "If there are credentials → Device activation has been performed" AND "If device activation has been performed → There are credentials".

This isn't the first time this has come up, and it won't be the last. :yum:

Reply:
Ah, I see! I hadn't encountered that shorthand before. Easy enough to revert that change.